xml/System.Runtime.CompilerServices/CallerArgumentExpressionAttribute.xml

```diff
@@ -19,7 +19,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Allows capturing of the expressions passed to a method.</summary>
     <remarks>To be added.</remarks>
```

Comment:
since we are not adding remarks, can

```
<remarks>To be added.</remarks>
```
stay here or gets removed?

Reply:
Stays here. It doesn't appear in the doc builds, and it makes it easier for us to identify sections with no content.